app/scripts/metamask-controller.js

```diff
@@ -7742,6 +7795,23 @@ export default class MetamaskController extends EventEmitter {
     );
   }
 
+  async _notifySolanaAccountChange(origin, accountAddress) {
+    this.notifyConnections(
+      origin,
+      {
+        method: NOTIFICATION_NAMES.walletNotify,
+        params: {
+          scope: MultichainNetworks.SOLANA,
+          notification: {
+            method: NOTIFICATION_NAMES.accountsChanged,
```

Comment:
I think we just use the `metamask_accountsChanged` event since this really bespoke and specific to our wallet's Wallet Standard interface needs 🤷 ?